### ISSUE TITLE
Error: Could Not Rename

### ISSUE THREAD

**Issue by rmisko11** [tldr-pages/tldr-c-client#24]
## Reproduction Rate
<How often could you reproduce this issue, how often do you tried?>
always, a lot

## Steps to Reproduce
<Exact steps to reproduce the issue>
tldr -u

## Result
<What is the result of this?>
e.g. Error: Could Not Rename: /tmp/tldrmkhmBO/tldr-master to /home/roger/.tldrc/tldr-master/

## Expected Result
<What is the expected result?>
File renamed and moved

## Additional Information
<Which version? Which operating system? Any other information?>

tldr v1.3.0 (v1.3.0-12-g0e54281)
Linux Kilo 4.10.1-041001-generic #201702260735 SMP Sun Feb 26 12:36:48 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux

/tmp and /home  are in separate partitions which maybe why rename reports the following error:

Invalid cross-device link

**Comment by iguessthislldo** [comment]
This is what #21 was meant to solve, unfortunately, it has yet to be accepted. If you wanted this to work at the moment you could install my fork or, maybe easier, use @12345ieee's workaround in the PR comments which manually puts the tldr files in the correct place.

**Comment by rmisko11** [comment]
Thanks for the feedback.

Sorry, I don't know git-hub very well, I just saw the 5 issues and didn't
see #21.

Anyway, for my quick and dirty work around I just replaced the rename with
the system command using mv.

**Comment by SethFalco** [comment]
Closing as a duplicate of the following:
* https://github.com/tldr-pages/tldr-c-client/issues/95

Progress on this has been made by a contributor here:
* https://github.com/tldr-pages/tldr-c-client/pull/100

---

I recognize this one is older and had more thumbs up. I opted to close this one, however, as the more recent issue is better formatted.
